### ISSUE TITLE
Videos are slow to play or freeze during play.

### ISSUE THREAD

**Issue by gdenne** [christianselig/apollo-bugs#1916]
**Describe the bug**
Videos are slow to play or freeze during play. This has happened in various Apollo releases over the years. It's in the current version again. Sigh. Note, the same posts have no issues playing in the official reddit app. They play instantly without interruption. If I switch from Apollo to official reddit and back again, the videos play perfectly in reddit and are freezing and flaky in Apollo.

**To Reproduce**
1. Click post with video.
2. Wait for video to start playing.
3. Video stuck. Will not play. 
4. Cannot view post.

**iOS version**: 15.7.1
**Apollo version**: 1.14.8
**iOS device**: iPad 9th gen
**Jailbroken**: No
**Reproducible:** Yes

**Expected behavior**
Video and GIF posts play on open and do not freeze.

**Screenshots**
If possible, add screenshots to help show the problem.

**Video**
If possible, attach a video to help show the problem. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.

**Additional context**

**Comment by huyz** [comment]
Yeah I’ve been putting up with videos that don’t play once you click on them and try to rewind to the beginning, for years. So frustrating I’ve started switching back to the official app, even with all the ads
